openverse_catalog/dags/common/storage/media.py

```diff
@@ -6,6 +6,7 @@
 
 from common.extensions import extract_filetype
 from common.licenses import is_valid_license_info
+from common.loader import provider_details as prov
```

Comment:
Nit: I think I personally prefer the full `provider` here instead of `prov` because it's clearer, but it doesn't matter much.

Reply:
I like the full `provider` here, but it's `prov` in all of the provider scripts, so I just used this for consistency (so that I don't have to change what I copy-pasted :) ) Do you think we should open an issue to replace `provider` with `prov`, @zackkrida?

Reply:
+1 to using the full name rather than an alias 😄 I think an issue for that would be excellent! It'd be a solid "good first issue" as well